frontend/src/components/chatInput/useChatInput.ts

```diff
@@ -1,28 +1,32 @@
-import { ChangeEvent, FormEvent, useState } from "react";
+import { useFormik } from "formik";
+import * as Yup from "yup";
 
-const useChatInput = () => {
-    const [currentInputValue, setCurrentInputValue] = useState("");
+export const messageRegExp =
```

Comment:
use the same regexp as in api:
/^(https?):\/\/(?:www\.)?[a-zA-Z]{2,}\.[a-z]{2,5}(?!.*(\/{2}|https?|:|\.| )).*$/